source/content/github-application.md

```diff
@@ -0,0 +1,84 @@
+---
+title: GitHub Application (Private Beta)
+description: Pantheon's GitHub Application handles moving code from individual GitHub repositories to individual Pantheon sites.
+tags: [continuous-integration, workflow, D8, D9, D10]
+contributors: [stevector]
+contenttype: [doc]
+innav: [true]
+categories: [automate, workflows]
+cms: [drupal, wordpress]
+audience: [development]
+product: [terminus]
+integration: [--]
+reviewed: "2025-04-15"
+
+---
+
+Pantheon's GitHub Application directly integrates a GitHub repository with a Pantheon site.
+This allows you to use GitHub as your code repository while still using Pantheon to run your site.
+
+The GitHub Application is currently in private Beta. [Request access for your Pantheon workspace here](https://docs.google.com/forms/d/e/1FAIpQLSf0vYrRbPQBxR-hT8kGJ4bEdYPtpkTtfDvPM89xD2dNZeqLqA/viewform).
+
+Once enabled, this application accommodates a pull request workflow where a Multidev environment is created for each pull request. This allows you to test the code in the pull request before merging it.
+
+![Deploying a PR to a Pantheon Multidev](../images/github-app/diagram--deploying-pr.png)
+
+Merging pull requests, or pushing code to the main branch of the GitHub repository, will automatically deploy the code to the Pantheon development environment.
+
+![Deploying main to Pantheon](../images/github-app/diagram--deploying-main.png)
+
+
+## Creating a new site
+
+Once accepted into the private Beta, you will be given access to a private Terminus plugin to enable site creation with the GitHub Application.
+
+## Try the pull request workflow
+
+To get comfortable using the workflow, you might want to do a few practice pull requests.
+
+First, after the site is created, install the CMS (WordPress or Drupal) within the Dev environment.
+Then create the Test and Live environments.
+
+### Make a pull request
+
+Make a local clone of the GitHub repository and make a change to the code.
+For example, you could add your favorite plugin or module.
+
```

Comment:
```suggestion

\```bash
git clone git@github.com:<your organization name>/<your site id>.git
\```

```

not sure how to put a code block inside a GitHub suggestion. Leading `\`s need to be stripped.